src/lib/jbuild

```diff
@@ -6,4 +6,4 @@
   (ocamlopt_flags (:standard -w +a-4-6-30-32-39-41-42-44-48-60 -warn-error +a -open Base -O3))
   (public_name tt)
   (libraries ())
-  (preprocess (pps (ppx_deriving.std ppx_jane ppx_let ppx_driver.runner)))))
```

Comment:
It turns out that `ppx_let` is redundant here because it's included with `ppx_jane`. You can see the list of other ppx that `ppx_jane` enables [here](https://github.com/janestreet/ppx_jane/blob/master/kernel/src/jbuild#L7-L23).